api/organisations/utils.py

```diff
@@ -0,0 +1,20 @@
+def dict_to_class(dictionary):
```

Comment:
This function is super ugly (and untested!), we cannot abuse python like this! Can we rethink this? I assume it's because we're trying to coerce a dictionary to a chargebee.Subscription model. 

If so, we should just create our own dataclass perhaps which we can create from a chargebee.Subscription model or a dictionary.

Reply:
Function removed. The approach changed to use `dict` instead of a class as the type of the `subscription` param.